doc/architecture/index.rst

```diff
@@ -68,4 +82,4 @@ Architecture
     ledger
     raft_tla
     node_to_node
-    indexing
\ No newline at end of file
+    indexing
```

Comment:
```suggestion
    indexing
    tls_internals
    quic_internals
```

This `toctree` directive produces the navigation tree on the left, so if new files aren't explicitly added they won't be easily discoverable.